webpack.config.js

```diff
@@ -202,6 +203,17 @@ const webpackConfig = {
 			filename: 'assets.json',
 			path: path.join( __dirname, 'server', 'bundler' ),
 		} ),
+		new CircularDependencyPlugin( {
+			// exclude detection of files based on a RegExp
+			exclude: /node_modules/,
+			// add errors to webpack instead of warnings
+			failOnError: false,
+			// allow import cycles that include an asyncronous import,
+			// e.g. via import(/* webpackMode: "weak" */ './file.js')
+			allowAsyncCycles: false,
+			// set the current working directory for displaying module paths
+			cwd: process.cwd(),
+		} ),
```

Comment:
maybe we could merge this with a flag so that it wouldn't affect most builds except for those where people are trying to work on de-cycling Calypso.

do you know this plugin affects build times?

Reply:
flag sounds like a plan. 

it didn't appear to materially affect build times in my limited testing. but if it's behind a flag, we can defer that investigation for a while.